runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerBox.java

```diff
@@ -303,12 +341,81 @@ private void buildItems()
 		itemContainer.repaint();
 	}
 
-	private static String buildToolTip(LootTrackerItem item)
+	private String buildToolTip(LootTrackerItem item)
 	{
 		final String name = item.getName();
 		final int quantity = item.getQuantity();
-		final long price = item.getPrice();
-		final String ignoredLabel = item.isIgnored() ? " - Ignored" : "";
-		return name + " x " + quantity + " (" + StackFormatter.quantityToStackSize(price) + ") " + ignoredLabel;
+		final long geValue = item.getGePrice();
+		final long haValue = item.getHaPrice();
+
+		StringBuilder builder = new StringBuilder();
+		builder
+			.append("<html>")
+			.append(name);
+
+		if (quantity > 1)
+		{
+			builder
+				.append(" x ")
+				.append(StackFormatter.quantityToStackSize(quantity));
+		}
+
+		if (item.isIgnored())
+		{
+			builder.append(" - Ignored");
+		}
+
+		if (valueType == LootTrackerValueType.BOTH)
+		{
+			builder
+				.append("<br/>")
+				.append("GE: ")
+				.append(buildTooltipItemValueText(geValue, quantity))
+				.append("<br/>")
+				.append("HA: ")
+				.append(buildTooltipItemValueText(haValue, quantity));
+		}
+		else
+		{
+			long value = (valueType == LootTrackerValueType.GRAND_EXCHANGE) ? geValue : haValue;
+
+			builder
+				.append("<br/>")
+				.append(buildTooltipItemValueText(value, quantity));
+		}
+
+		builder.append("</html>");
+
+		return builder.toString();
+	}
+
+	private String buildTooltipItemValueText(long value, int quantity)
+	{
+		if (quantity > 1)
+		{
+			return new StringBuilder()
+				.append(StackFormatter.quantityToStackSize(value))
+				.append(" gp (")
+				.append(StackFormatter.quantityToStackSize(value / quantity))
+				.append(" ea)")
+				.toString();
+		}
+		else
```

Comment:
Having else here is unnecessary

Reply:
de0366819507da1b1d7aa15cc39f99176f5f9c62